db/columns/operations/alter.py

```diff
@@ -283,21 +284,29 @@ def _batch_alter_table_rename_columns(table_oid, column_data_list, connection, e
 
 
 def batch_alter_table_drop_columns(table_oid, column_data_list, connection, engine):
-    table = reflect_table_from_oid(
-        table_oid,
-        engine,
-        connection_to_use=connection,
-        # TODO reuse metadata
-        metadata=get_empty_metadata(),
-    )
-    ctx = MigrationContext.configure(connection)
-    op = Operations(ctx)
-    with op.batch_alter_table(table.name, schema=table.schema) as batch_op:
-        for column_data in column_data_list:
-            column_attnum = column_data.get('attnum')
-            if column_attnum is not None and column_data.get('delete') is not None:
-                name = get_column_name_from_attnum(table_oid, column_attnum, engine=engine, metadata=get_empty_metadata(), connection_to_use=connection)
-                batch_op.drop_column(name)
+    """
+    Drop the given columns from the given table.
+
+    Args:
+        table_oid: OID of the table whose columns we'll drop.
+        column_data_list: The attnums of the columns to drop.
```

Comment:
Doesn't seem accurate; sounds like it's an iterable of attnums, but it's actually an iterable of dict-likes.

Reply:
You're correct; Fixed now.